docs/admin/multiple-schedulers.md

```diff
@@ -0,0 +1,214 @@
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: my-scheduler
+  namespace: kube-system
+spec:
+  replicas: 1
+  selector:
+    matchExpressions:
+    - {key: tier, operator: In, values: [control-plane]}
+    - {key: component, operator: In, values: [scheduler]}
+    - {key: version, operator: In, values: [second]}
+  template:
+    metadata:
+      labels: 
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      containers:
+      - command: [/usr/local/bin/kube-scheduler, --address=0.0.0.0,
+                  --scheduler-name=my-scheduler]
+        image: gcr.io/my-gcp-project/my-kube-scheduler:1.0
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+          initialDelaySeconds: 15
+        name: kube-second-scheduler
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10251
+        resources:
+          requests:
+            cpu: '0.1'
+        securityContext:
+          privileged: false
+        volumeMounts: []
+      hostNetwork: false
+      hostPID: false
+      volumes: []
+```
+
+An important thing to note here is that the name of the scheduler specified as an argument to the scheduler command in the container spec should be unique. This is used to map pods to schedulers.
+
+Please see the [kube-scheduler documentation](http://kubernetes.io/docs/admin/kube-scheduler/) for detailed description of other command line arguments.
+
+### 3. Run the second scheduler in the cluster
+
+In order to run your scheduler in a Kubernetes cluster, just create the deployment specified in the config above in a Kubernetes cluster:
+
+```sh
+$ kubectl create -f my-scheduler.yaml
+```
+
+Verify that the deployment and hence the scheduler pod is running:
+
+```sh
+$ cluster/kubectl.sh get pods --namespace=kube-system
+NAME                                           READY     STATUS    RESTARTS   AGE
+....
+kube-scheduler-lnf4s-4744f                     1/1       Running   0          2m
+...
+```
+
+You should see a "Running" kube-scheduler pod. It is also important to note that the default scheduler managed by Kubernetes runs in a container but not in a pod, so you don't see it in the pods list here.
+
+### 4. Specify schedulers for pods
+
+In order to schedule a given pod using a specific scheduler, we specify the name of the scheduler as an annotation in that pod spec. Let's look at three examples.
```

Comment:
I just added this sentence before "In order to schedule ...". It reads Ok to me. Please take a look again.